learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -167,19 +194,21 @@ service /covid19 on new graphql:Listener(9000) {
 }
 ```
 
-In the above code snippet, the path of this service is defined as `/covid19`. If you want to host the service on the
-root, you can remove the path as the following code snippet:
+In this code:
+- The path of this service is defined as `/covid19`. If you want to host the service on the
```

Comment:
Why adding a bullet here?

Reply:
It was added to be consistent with the other `In this code:` sections. @anupama-pathirage

Reply:
Removed the bullet point.